ZenPacks/zenoss/Microsoft/Windows/datasources/ShellDataSource.py

```diff
@@ -753,8 +751,8 @@ def params(cls, datasource, context):
         if hasattr(context, 'cluster_node_server'):
             owner_node, _ = context.cluster_node_server.split('//')
             try:
-                owner_node_ip = getHostByName(owner_node)
-            except gaierror:
+                owner_node_ip = context.device().clusterhostdevicesdict[owner_node]
+            except Exception:
                 pass
```

Comment:
Its more efficient to set the default value to None for owner_node_ip, then do a context.device().clusterhostdevicesdict.get(...) . Having it set to blank string has side effects.